test/Microsoft.ML.Tests/OnnxConversionTest.cs

```diff
@@ -1310,22 +1310,25 @@ public void WordTokenizerOnnxConversionTest()
             IEstimator<ITransformer>[] pipelines =
             {
                 mlContext.Transforms.Text.TokenizeIntoWords("Tokens", "Text", new[] { ' ' })
-                                .Append(mlContext.Transforms.Conversion.MapValueToKey("Tokens"))
-                                .Append(mlContext.Transforms.Text.ProduceNgrams("NGrams", "Tokens",
-                                            ngramLength: ngramLength,
-                                            useAllLengths: useAllLength,
-                                            weighting: weighting)),
+                .Append(mlContext.Transforms.Conversion.MapValueToKey("Tokens"))
```

Comment:
Any particular reason the formatting was changed? Usually there is a small offset to indicate line continuation.

Reply:
No, I noticed that line 1321 was not aligned with the rest. I'll add the offsets back though and just align that line.

Reply:
nit: It looks like the offset for line 1313 is still wrong. It seems to be left aligned with the previous line.